### PR TITLE
Feature/quality gate filter

### PR DESCRIPTION
**Pull request by alexalvess**
- made the rule
- throw an error if the condition not attend
- the default value is 0%

### CODE REVIEW

File: buildAndReleaseTask/environments/environment.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ export const environment = {
     swaggerJsonPath: '/swagger/v1/swagger.json',
     testResultPath: 'C:\\Users\\alexa\\Downloads',
```
Comment (raphaelbs) [inline buildAndReleaseTask/environments/environment.ts:5]: :eyes:

**Review by raphaelbs — APPROVED** [review]
Consider adding documentation explaining why this rule has been added and how to use it!

File: buildAndReleaseTask/environments/environment.ts (hunk 1 above)
Comment (alexalvess) [reply buildAndReleaseTask/environments/environment.ts:5]: I will build a wiki for this

**Comment by alexalvess** [comment]
> Consider adding documentation explaining why this rule has been added and how to use it!

**Comment by alexalvess** [comment]
> Consider adding documentation explaining why this rule has been added and how to use it!

I will build a wiki for this

**Pull request closed without merging**